portfolio/src/main/webapp/index.html

```diff
@@ -4,10 +4,11 @@
     <meta charset="UTF-8">
     <title>Lexi Delorey</title>
     <link rel="stylesheet" href="style.css">
+    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyBNcGaKmWLRb_rvQf-hIAVSChe6TkZu-1Y"></script>
```

Comment:
FYI: In general it's best to avoid checking API keys or other secrets into your repository. In this case it's hard to avoid exposing the key itself (because anyone who opens your webpage can inspect the `<script>` tag), but there are [other security precautions](https://developers.google.com/maps/api-key-best-practices#api_key_table) you can take.

I've seen friends get bitten by this: attackers scraped paid API keys from public GitHub repositories and used them to make many thousands of requests, putting my friends on the hook for large charges from the API provider. Luckily they were able to get the fraudulent charges removed, but the incidents were still very scary.

If your API key is not on a paid (premium) plan, don't worry about changing anything here--just be aware of the risks in the future.